README.md

```diff
@@ -22,3 +22,10 @@ your `.vimrc`.
 let g:fubitive_domain_pattern = 'code\.example\.com'
 let g:fubitive_domain_context_path = 'bitbucket'
 ```
+
+By default, fubitive will assume `https://` when building URLs. 
+To change this, set the `fubitive_default_protocol` variable:
```

Comment:
I'd just prefix it with `g:` here, or write "...set the global `fubitive_default_protocol` variable:"

Reply:
Done!